early-feedback.md

```diff
@@ -0,0 +1,2 @@
+We are collecting data on the user experience of the Coq Bug Minimizer.
+If you haven't already filled the survey for this PR, please fill out [our short survey](@SURVEY_URL@)!
```

Comment:
What about adding some emphasis on the fact that we are collecting feedback on each PR separately:

```suggestion
If you haven't already filled the survey **for this PR**, please fill out [our short survey](@SURVEY_URL@)!
```

or:

```suggestion
If you haven't already filled the survey *for this PR*, please fill out [our short survey](@SURVEY_URL@)!
```